ddp-workspace/projects/ddp-rgp/src/app/constants/study-messages.ts

```diff
@@ -106,9 +102,12 @@ export const studyMessagesConfiguration: StudyMessageConfiguration[] = [
   },
   {
     workflowKey: WorkflowKey.AcceptanceStatus,
-    condition: workflow =>
-      workflow.status === AcceptanceStatus.Accepted ||
-      workflow.status === AcceptanceStatus.MoreInfoNeeded,
+    condition: workflow => [
+        AcceptanceStatus.Accepted,
+        AcceptanceStatus.MoreInfoNeeded,
+        AcceptanceStatus.NmiToAccept,
+        AcceptanceStatus.NotAccepted,
+      ].includes(workflow.status as AcceptanceStatus),
     additionalCondition: workflows =>
       !!workflows.find(workflow => workflow.workflow === WorkflowKey.AcuityAppointmentDate && workflow.status) &&
       !workflows.find(workflow => workflow.workflow === WorkflowKey.DateOfConsentCall && workflow.status),
```

Comment:
Looks like we need to remove the condition that depends on `DateOfConsentCall` (see DDP-6445).